src/organisms/tables/ComparisonTable/index.js

```diff
@@ -42,7 +42,7 @@ class TableHeader extends React.Component {
     const childrenArray = React.Children.toArray(children);
 
     if (showMissing && tableItems === 1 && childrenArray.length === 1) {
-      return childrenArray.concat([ <Col /> ]);
+      return childrenArray.concat([ <Col key={1} /> ]);
```

Comment:
i wouldn't hardcode this `key`. I would set it to `childrenArray.length`

Reply:
done